uva-2.config

```diff
@@ -0,0 +1,18 @@
+uva-2.bar_field = 955
+uva-2.bar_field_mult = 1 
+uva-2.bar_sfield = i
+uva-2.campus_code = uva
```

Comment:
This doesn't conflict with the first uva config?

Reply:
I guess my real question is, will uva-2 files go in a separate directory from the old uva?

I'm not clear if this config campus_code is the same as the "campus code" used in process_indata to apply the special sauce.

Reply:
The code in the config's campus_code is used to prefix the bib record's system ID, to form the "sdr" number. I believe the campus code used in process_indata is the 'name' of the stream, which would be uva-2. Should we double-check? The tests on dev have gone correctly.

Reply:
Okay thanks! That answers my question, should be good to go.

Reply:
Just a bit more - my notes in ticket 21371 indicate that process_indata defines a 'campus' variable according to the name of a directory. So I believe we're all set, thx!